EIPS/eip-4950.md

```diff
@@ -0,0 +1,89 @@
+---
+eip: 4950
+title: Entangled Tokens
+description: ERC-721 extension with two tokens minted that are tied together
+author: Victor Munoz (@victormunoz)
+discussions-to: https://ethereum-magicians.org/t/entangled-tokens/8702
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-28
+requires: 721
+---
+
+## Abstract
+This EIP standardizes an interface for creating entangled pairs of tokens. An entangled pair of tokens give his owners the right to operate with this common wallet.
+
+## Motivation
+The motivation is to provide an easy way to share a wallet through NFTs. So that the act of buying an NFT (in a marketplace) gives the buyer the privilege to have access to a given wallet. This wallet could have budget in many tokens, or even be the propietary of other NFTs.

```

Comment:
Grammar and spelling, not sure if I picked the right word:

```suggestion
The motivation is to provide an easy way to share a wallet through NFTs, so that the act of buying an NFT (in a marketplace) gives the buyer the privilege to have access to a given wallet. This wallet could have budget in many tokens, or even be the property of other NFTs.

```